scss/control.scss

```diff
@@ -285,8 +285,18 @@
 	position: relative;
 }
 
+// visually hidden fields
 
-
+.Select-hidden {
```

Comment:
Is this file supposed to have the same content as the `less` file? If yes, its missing a bunch of changes.

Reply:
PS: this is resolved. I miss GitLab's "Mark resolved" feature here.